packages/protocol/package.json

```diff
@@ -43,7 +43,6 @@
     "@0x/sol-profiler": "^3.0.0",
     "@0x/sol-trace": "^2.0.16",
     "@0x/subproviders": "^5.0.0",
-    "@openzeppelin/upgrades": "^2.8.0",
```

Comment:
why this one? (it doesn't seem to be part of the same issue)

Reply:
This line was duplicated (see below) and when I use `yarn upgrade-interactive` it rewrote it (and others).

Reply:
Ah, you are right!